### DISCUSSION TITLE
Can't connect to database

### DISCUSSION THREAD

**Discussion by musubipapi** [prisma/prisma#8026] [Q&A]
I have been running into the following error when connecting to my database:

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "railway", schema "public" at "containers-us-west-7.railway.app:8048"

Error: Database error
Error querying the database: db error: ERROR: invalid string in message
   0: sql_migration_connector::flavour::postgres::sql_schema_from_migration_history
             at migration-engine/connectors/sql-migration-connector/src/flavour/postgres.rs:367
   1: migration_core::api::DevDiagnostic
             at migration-engine/core/src/api.rs:89

```

It works fine on MacOS but whenever I try to run it on WSL2 this error occurs.

Any help would be appreciated!

**Comment by janpio** [comment]
That's a weird one. How can we reproduce?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
